src/app/containers/Video/index.jsx

```diff
@@ -60,10 +78,39 @@ const VideoContainer = ({ blocks }) => {
               {JSON.stringify(metadata)}
             </script>
           }
+          <script type="text/javascript">
+            {`
+              function mediaPlayerSetup(container) {
+                  require(['bump-4'], (bump) => {
+                    var mediaPlayer = bump.player(
+                      document.getElementById('mediaPlayer'),
+                      ${JSON.stringify(mediaPlayerSettings)});
+                    mediaPlayer.load();
+                  });
+              }
+            `}
+          </script>
+          <script type="text/javascript">
+            {`
+              function initialiseRequires() {
+                  requiredScripts = {
+                    "bump-4":"//emp.bbci.co.uk/emp/bump-4/bump-4"
+                    };
+                  require({ paths: requiredScripts, waitSeconds: 30 });
+                  mediaPlayerSetup();
+              }
+            `}
+          </script>
+          <script
+            onLoad="initialiseRequires()"
```

Comment:
This `onLoad` chaining, along with `mediaPlayerSetup();` on line 100 is to get around the asynchronicity of script loading in helmet. 
React-helmet team have [their own solution in the comments of this issue](https://github.com/nfl/react-helmet/issues/419) and I started to try to use it _but_ as the example SMP embedding code which I've used here is designed to be run client side I chose the multi-script tag and `onLoad` chaining approach.